mars/services/task/execution/ray/executor.py

```diff
@@ -462,3 +549,142 @@ def _get_subtask_output_keys(chunk_graph: ChunkGraph):
             else:
                 output_keys[chunk.key] = 1
         return output_keys.keys()
+
+    async def _get_ready_objects(self):
+        total = len(self._cur_stage_output_object_refs)
+        if total > 0:
+            ready_objects, _ = ray.wait(
+                self._cur_stage_output_object_refs,
```

Comment:
We should only wait for the unfinished object refs, not the entire stage output object refs.

Reply:
Good suggestion, thanks.